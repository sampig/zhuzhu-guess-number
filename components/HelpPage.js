import React from 'react'
import { StyleSheet, Text, View } from 'react-native'
import i18n from 'util/i18n-utils'

const styles = StyleSheet.create({
  container: {
    flex: 1,
    display: 'flex',
    marginLeft: 16,
    marginRight: 16,
    alignItems: 'flex-start',
    justifyContent: 'flex-start',
  },
  title: {
    fontSize: 30,
    marginTop: 25,
    marginBottom: 25,
  },
  createTime: {
    fontSize: 18,
  },
})

/**
 * Component to a to-do detail page.
 * @returns {node} - React node
 * @constructor
 */
function HelpPage () {
  return (
    <View style={styles.container}>
      <Text style={styles.title}>{i18n.t('gameName')}</Text>
      <Text style={styles.createTime}>{i18n.t('gameDescription')}</Text>
    </View>
  )
}

export default HelpPage
