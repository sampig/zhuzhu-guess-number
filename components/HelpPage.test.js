import React from 'react'
import renderer from 'react-test-renderer'
import HelpPage from './HelpPage'


describe('The HelpPage component', () => {
  it('renders without crashing', () => {
    const render = renderer.create(<HelpPage />).toJSON()
    expect(render).toMatchSnapshot()
  })
})
