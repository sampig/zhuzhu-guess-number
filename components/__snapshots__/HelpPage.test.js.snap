// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The HelpPage component renders without crashing 1`] = `
<View
  style={
    Object {
      "alignItems": "flex-start",
      "display": "flex",
      "flex": 1,
      "justifyContent": "flex-start",
      "marginLeft": 16,
      "marginRight": 16,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 30,
        "marginBottom": 25,
        "marginTop": 25,
      }
    }
  >
    zhuzhu Guess Number
  </Text>
  <Text
    style={
      Object {
        "fontSize": 18,
      }
    }
  >
    Have 10 chances to guess the 4-digit number (each digit is different). For example, "1âœ“ 1X" means that there is 1 digit that is totally correct, there is 1 totally wrong digit, and 2 other digits are contained but in wrong position.
  </Text>
</View>
`;
